assets/js/main.js

```diff
@@ -290,6 +290,31 @@ function elasticSearch(query, callback) {
 
 var searchSelectionId = 0;
 var searchListingLength = 0;
+function isNumeric(n) {
+    return !isNaN(parseFloat(n)) && isFinite(n);
+}
+function categoriseResult(post) {
+    if (!post.url_path_dir1 || !post.url_path_dir1.raw) {
+        return ['Home'];
+    }
+    if (isNumeric(post.url_path_dir1.raw)) {
+        var tags = ['Post'];
+        if (post.title.raw.startsWith('Conversations with')) {
+            tags.push('Conversations');
+        }
+        if (post.title.raw.startsWith('Reflections with')) {
+            tags.push('Reflections');
+        }
+        if (post.title.raw.startsWith('Insights on')) {
+            tags.push('Insights');
+        }
+        return tags;
+    }
+    var page_type = post.url_path_dir1.raw
+        .replace('-', ' ')
+        .replace(/(^\w{1})|(\s+\w{1})/g, (letter) => letter.toUpperCase());
```

Comment:
Consider using https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform instead.

Reply:
Settled it using tailwind
https://tailwindcss.com/docs/text-transform